app/views/consignment_references.scala.html

```diff
@@ -32,7 +32,7 @@
     @helper.form(controllers.movements.routes.ConsignmentReferencesController.saveConsignmentReferences(), 'autoComplete -> "off") {
         @helper.CSRF.formField
 
-        @components.back_link(controllers.routes.ChoiceController.displayPage())
+        @components.back_link(controllers.ileQuery.routes.IleQueryController.submitQueryForm())
```

Comment:
Why is this so? This way after choice page if I realise that I need to d osth else, I have to go through the whole ILE query again. I am quite sure this is not expected behaviour.

Reply:
After a short discussion in the exports-design channel, I will revert this and create a back button pointing to the query page.